src/components/bottom-widget.tsx

```diff
@@ -24,9 +24,9 @@ import TimeWidgetFactory from './filters/time-widget';
 import AnimationControlFactory from './common/animation-control/animation-control';
 import AnimationControllerFactory from './common/animation-control/animation-controller';
 import {ANIMATION_WINDOW, DIMENSIONS, FILTER_TYPES} from '@kepler.gl/constants';
-import {getIntervalBins} from 'utils/filter-utils';
+import {getIntervalBins} from 'reducers/layer-utils/filter-utils';
```

Comment:
why is this in `layer-utils/filter-utils`? can't this just be under `reducers/filter-utils`?